src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    public Colori getRandomColor() {
```

Comment:
The method does not have an access modifier. It's recommended to include an access modifier to make the visibility of the method explicit. This is a code style issue.